hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -812,43 +811,54 @@ private static DNAction getAuditAction(Type cmdType) {
     }
   }
 
+  private static final String AUDIT_PARAM_CONTAINER_ID = "containerID";
```

Comment:
By convention the private static final constant fields are declared at the start of the class. Please move these declarations to around line 94.

I think the audit related methods and all these constants could be moved to a separate class/method to make HddsDispatcher shorter. What do you think @adoroszlai ?

Reply:
Thanks @Galsza  for the comments and review.
I've updated it.